packages/otelbin/src/components/monaco-editor/otelCollectorConfigValidation.ts

```diff
@@ -156,28 +168,30 @@ export function customValidate(
 					message: errorMessage,
 				};
 				errorMarkers.push(errorMarker);
-				totalErrors.customErrors?.push(errorMarker.message + " " + `(line ${line})`);
+				totalErrors.customErrors?.push(errorMarker.message + " " + `(Line ${line})`);
 			}
 		});
-		mainItems?.forEach((item) => {
-			if (!serviceItems.some((serviceItem) => serviceItem.source === item.source)) {
-				const errorMessage = `${capitalize(key)} "${item.source}" is unused.`;
-				const { line, column } = findLineAndColumn(configData, item.offset);
-				const endColumn = column + (item.source?.length ?? 0);
 
-				const errorMarker = {
-					startLineNumber: line ?? 0,
-					endLineNumber: 0,
-					startColumn: column ?? 0,
-					endColumn: endColumn,
-					severity: 4,
-					message: errorMessage,
-				};
-				errorMarkers.push(errorMarker);
+		!isServerSideValidationEnabled &&
```

Comment:
**Suggestion:** This is a really uncommon code pattern outside of JSX. Please consider using regular `if` statements outside of JSX.

Reply:
@bripkens sure, thank you